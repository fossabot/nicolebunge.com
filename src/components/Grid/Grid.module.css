:root {
  --grid-max-width: 1440px;
}

.grid {
  margin-left: auto;
  margin-right: auto;
  max-width: var(--grid-max-width);
  width: 90%;
}
