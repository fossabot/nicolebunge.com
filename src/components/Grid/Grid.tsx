import clsx from 'clsx';
import React, { HTMLAttributes, ReactNode } from 'react';
import * as styles from './Grid.module.css';

export interface GridProps extends HTMLAttributes<HTMLDivElement> {
  children: ReactNode;
}

function Grid(props: GridProps): JSX.Element {
  const { className, ...otherProps } = props;

  return <div className={clsx(className, styles.grid)} {...otherProps} />;
}

export default Grid;
