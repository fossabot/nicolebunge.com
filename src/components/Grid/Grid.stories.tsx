import Grid from '.';

export default {
  component: Grid,
};

export const Default = {
  args: {
    children: 'Grid',
  },
};
