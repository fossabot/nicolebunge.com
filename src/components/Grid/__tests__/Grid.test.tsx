/* eslint-disable import/no-extraneous-dependencies */

import { render } from '@testing-library/react';
import React from 'react';
import Grid from '..';

describe('Grid component', () => {
  it('renders correctly', () => {
    const { container } = render(<Grid>Grid</Grid>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
