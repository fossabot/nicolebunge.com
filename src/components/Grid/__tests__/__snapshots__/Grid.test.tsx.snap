// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid component renders correctly 1`] = `
<div
  class=""
>
  Grid
</div>
`;
